src/dns/nuke/index.js

```diff
@@ -48,7 +49,14 @@ module.exports = function nukeDNS(parms, callback) {
         deleteCNAMERecords,
         deleteHostedZone,
         deleteCertificate,
-      ], callback)
+      ],
+      function done(err, results, callback) {
+        if (err) callback
```

Comment:
errbacks (where the first param is an error) should not (and mostly will not) pass a callback as a third param. you'll want to both remove that third param and ensure the `if (err) callback(err)` and likewise the else actually invokes the callback on 57 w parens `callback()`